_server/tests/unit/PurchaseOrdersRecordRepositoryTest.cpp

```diff
@@ -0,0 +1,93 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <vector>
+#include <iostream>
+
+#include "mocks/DatabaseManagerMock.hpp"
+#include "Database/PurchaseOrdersRecordRepository.hpp"
+#include "ServerException.hpp"
+
+#include "common/Entities/PurchaseOrderRecord.hpp"
+#include "common/Keys.hpp"
+#include "common/SpdlogConfig.hpp"
+
+using ::testing::_;
+using ::testing::Return;
+
+class PurchaseOrderRecordRepoTest : public ::testing::Test 
+{
+protected:
+    std::unique_ptr<PurchaseOrderRecordRepository> purchOrdRecRepo;

```

Comment:
Do not make names short. Keep it `purchaseOrderRecordRepository`

Reply:
done